### PR TITLE
Locale/fr

### PR DESCRIPTION
**Pull request by LukyVj**

### CODE REVIEW

File: public/locales/fr/translate.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "intro": {
+    "text": [
+      "Dans ce jeu, il y a deux rôles: le détective et le menteur. A chaque tours, vous allez alterner entre ces deux rôles.",
```
Comment (Haroenv) [inline public/locales/fr/translate.json:4]:
```suggestion
      "Dans ce jeu, il y a deux rôles: le menteur et le détective. A chaque tours, vous allez alterner entre ces deux rôles.",
```

File: public/locales/fr/translate.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "intro": {
+    "text": [
+      "Dans ce jeu, il y a deux rôles: le menteur et le détective. A chaque tours, vous allez alterner entre ces deux rôles.",
```
Comment (Haroenv) [inline public/locales/fr/translate.json:4]:
```suggestion
      "Dans ce jeu, il y a deux rôles: le menteur et le détective. A chaque tour, vous allez alterner entre ces deux rôles.",
```

File: public/locales/fr/translate.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      "Le menteur aura deux articles Wikipedia au hasard qui lui seront présenté. Il ne lui sera permi de n'en lire qu'un seul.",
+      "Ensuite, le détective verra le titre des deux pages, et aura l'opportunité de poser n'importe quelle question à propos des deux pages.",
+      "Est-ce que le détective pourra trouver laquelle de ces deux pages le menteur a inventé?"
+    ],
+    "language": "<0>language: </0>",
+    "player-one": "<0>Premier joueur: </0>",
```
Comment (Haroenv) [inline public/locales/fr/translate.json:10]:
```suggestion
    "player-one": "<0>premier joueur: </0>",
```

File: public/locales/fr/translate.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "player-two": "<0>Second joueur: </0>",
```
Comment (Haroenv) [inline public/locales/fr/translate.json:11]:
```suggestion
    "player-two": "<0>second joueur: </0>",
```

why not "deuxième"?

File: public/locales/fr/translate.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    "player-identical": "Les joueurs ne peuvent pas avoir le même nom",
+    "start": "Commencer!",
+    "footer": "Ce jeu a été inspiré par <2>Two Of These People Are Lying</2> de Tom Scott et Matt Grey."
+  },
+  "game": {
+    "try-again": "essayez encore",
+    "choosing": "C'est maintenant au tour du menteur ({{liar}}). de lire a propos des deux sujets, s'il vous plaît, choisissez l'un des deux articles:",
```
Comment (Haroenv) [inline public/locales/fr/translate.json:18]:
```suggestion
    "choosing": "C'est maintenant au tour du menteur ({{liar}}) de lire a propos des deux sujets, s'il vous plaît, choisissez l'un des deux articles:",
```

File: public/locales/fr/translate.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    "done-reading": "J'ai fini de lire",
+    "preguessing": "S'il vous plaît, veuillez donner l'appareil à {{investigator}}",
+    "start-guessing": "Commencez à deviner",
+    "guessing": "C'est maintenant au tour du détective ({{investigator}}). Vous pouvez maintenant poser des questions à propos de ces articles. Une fois que vous pensez que vous avez trouvé quel article la personne a lu, faites une proposition!",
+    "scratchpad": "Voici un bloc-notes pour laisser vos notes: ",
+    "scratchpad-tips": "Il connaissait un détail intéressant, l'a t’il lu à l'instant? Ou est-ce une coïncidence… Écrivez le pour vous en souvenir",
+    "investigator-won": "Le détective, {{ investigator }} a gagné! {{ liar }} a bien lu à propos de {{ title }}",
+    "liar-won": "Le menteur, {{ liar }} a gagné! {{ liar }} a effectivement lu à propos de {{ title }}",
+    "read-articles": "Lire les articles suivant: ",
+    "start-next": "Commencer prochaine manche",
```
Comment (Haroenv) [inline public/locales/fr/translate.json:29]: you used "tours" in the start, which one fits better in both cases?

File: public/locales/fr/translate.json [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    "scores": "scores",
+    "stop-playing": "arrêter de jouer",
+    "final-score": "Le score final est:",
+    "did-read": "Les articles que vous avez lu étaient:",
```
Comment (Haroenv) [inline public/locales/fr/translate.json:33]:
```suggestion
    "did-read": "Les articles que vous avez lu étaient :",
```

not sure if is true anymore, but I thought there was a space before colon in French? (same for other ones)

File: public/locales/fr/translate.json (hunk 6 above)
Comment (LukyVj) [reply public/locales/fr/translate.json:29]: not sure actually, give me a sec, daddy to the rescue 😂

**Pull request merged**